website/project/views/node.py

```diff
@@ -724,6 +724,8 @@ def _view_project(node, auth, primary=False):
             'tags': [tag._primary_key for tag in node.tags],
             'children': bool(node.nodes),
             'is_registration': node.is_registration,
+            'is_retracted': node.is_retracted,
```

Comment:
@sloria @jmcarp -- do you think it's worth pulling these fields into a subdict? These flat dicts are getting hard to read.